components/eam/src/chemistry/mozart/mo_gas_phase_chemdr.F90

```diff
@@ -793,7 +795,7 @@ subroutine gas_phase_chemdr(lchnk, ncol, imozart, q, &
     call t_startf('aero_model_gasaerexch')
     call aero_model_gasaerexch( imozart-1, ncol, lchnk, delt, latndx, lonndx, reaction_rates, &
                                 tfld, pmid, pdel, mbar, relhum, &
-                                zm,  qh2o, cwat, cldfr, ncldwtr, &
+                                zm,  qh2o, cwat_liq, cldfr, ncldwtr, &
```

Comment:
This change does not come in with the SCREAM's Fortran P3 (via PR #5416). I don't feel like we need to keep using 'cwat' for SCREAM configuration and 'cwat_liq' for EAM configuration. @yunpengshan2014 and @crterai , what do you think? If need to have a distinction for SCREAM, we can use CPP macro if defined(SCREAM) to control.

Reply:
@wlin7 I'll need to remove this, since we want P3 to go in as Stealth. And then this change will need to go in as a separate PR.

Reply:
Oh, yes. Do need some treatment here to maintain as a stealth feature. Can we use the 'microp_scheme' value here, and let cwat_liq = cwat unless it is P3? That also means this change would not be activated for MG2.

Reply:
Yes, instead of removing, we could make it as if (microp_name == 'p3' & not(SCREAM)) then ... 
It will preserve the v2 results in master. However, it will also introduce a fair amount of code clutter, especially if there's agreement that this is something that needs to be changed. Interested in thoughts from @yunpengshan2014, @hwangacme, among others.

Reply:
If we want to keep the change for SCREAM config, it should be simpler. Comments from @PeterCaldwell will also be helpful. We want to preserve the SCREAM version of Fortran 90 P3. Other parts of Fortran EAM  codes needed by SCREAM can evolve with master (thus NBFB for scream config). This is not limited to the current P3 PR.

Reply:
None of this aerosol stuff is in the C++ code at all. And while F90 SCREAM *can* run with MAM, that capability is not supported in any sense of the word. Thus even though I don't really understand the ramifications of switching between total condensate and liquid condensate for gasaerexchange, I'm pretty sure this change will have no impact on anything SCREAM cares about. 

@polunma might be interested in knowing this change is happening since I think he's been using SCREAM with MAM.

Reply:
Thanks, @PeterCaldwell . The other PRs from v3atm should have no impact on SCREAM config either. Most are for aerosols and gas chemistry. Others will be going in as stealth features, including ZM enhancements, which SCREAM-LR can choose to enable or not to.

Reply:
For the two different versions of p3, is it possible to really treat them as separate microphysics packages?  You would choose one or the other and the EAM configure stage.

Reply:
I think v3 versus SCREAM p3 *is* being handled as 2 separate copies, Rob. This particular conflict was something outside of P3. I would love to see a diff of the v3 versus SCREAM versions of P3 though... they have a common ancestor and I have no idea how much they have diverged. Maybe it would be easy to merge them back into 1 copy?

Reply:
The differences in the interface and the main P3 module (micro_p3.F90) are quite substantial. The cleanest and least time consuming is to maintain two copies. The PR for alternating control is almost ready.

Reply:
I am changing this back to `cwat` and we will issue a new PR that specifically targets the aqueous chemistry change